src/main/scala/io/iohk/ethereum/jsonrpc/JsonRpcError.scala

```diff
@@ -0,0 +1,5 @@
+package io.iohk.ethereum.jsonrpc
+
+import org.json4s.JsonAST.JValue
+
+case class JsonRpcError(code: Int, message: String, data: Option[JValue])
```

Comment:
Should we incorporate in this PR handling the different errors with their respective codes? (as specified in http://www.jsonrpc.org/specification). Maybe we could create a separate task to handle this.
For example we currently fail to send a response when the request is: `{"jsonrpc":"2.0","method":"web3_clientVersion",params":[],"id":67}`